core/src/edu/asu/c3simulator/screens/ManagePanel.java

```diff
@@ -0,0 +1,54 @@
+package edu.asu.c3simulator.screens;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+
+import edu.asu.c3simulator.widgets.NavigationPanel;
+
+/**
+ * Creates a panel that allows for navigation through the management screens.
```

Comment:
Sounds like a method again. I want to know what ManagementPanel IS in this class javadoc